src/native/fs/fs_napi.cpp

```diff
@@ -1034,7 +1058,9 @@ struct Readfile : public FSWorker
             p += len;
         }
 
-        CHECK_CTIME_CHANGE(fd, _stat_res, _path);
+        if (!disable_ctime_check) {
+            CHECK_MTIME_CHANGE(fd, _stat_res, _path);
+        }
```

Comment:
Need to fix this to:
```suggestion
        if (disable_ctime_check) {
            CHECK_MTIME_CHANGE(fd, _stat_res, _path);
        } else {
            CHECK_CTIME_CHANGE(fd, _stat_res, _path);
        }
```

Reply:
Thank you.